module1/homework/grayscaleImages/test.cpp

```diff
@@ -95,6 +96,7 @@ TEST(compressionTests, ShouldCompressOneEighthLines) {
     expectBitmap(bitmap, 8);
 }
 
+
```

Comment:
what are those two new empty lines in this file?

Reply:
Artistic reasons :p.
Also to celebrate my new 2TB SSD.

Reply:
Typically such comment is valid if file was modified only with empty line. But normally we just ignore that.